test/keycloak-connect-web-spec.js

```diff
@@ -71,6 +73,62 @@ test('Should login with admin credentials', t => {
   });
 });
 
+test('Login should not change tokens when they are valid', t => {
+  t.plan(3);
+
+  page.get(app.port);
+
+  page.logInButton().click();
+  page.login('test-admin', 'password');
+
+  return page.events().getText().then(text => {
+    t.equal(text, 'Auth Success', 'User should be authenticated');
+    return page.output().getText().then(firstToken => {
+      page.logInButton().click(); // Invoke login for the second time, token shouldn't be changed
+      return page.output().getText().then(secondToken => {
+        t.equal(secondToken, firstToken, 'Token should not be changed as first session is still valid');
+        page.logOutButton().click();
+        return page.output().getText().then(text => {
```

Comment:
This test is not working for me in phantomjs. Me and @vmuzikar tried many things to make it work, however we were not successful. In chrome it works well.

Reply:
@mhajas could you please provide your error output? Just to make sure we're talking about the same thing.

Reply:
@abstractj Here is the output of my run: https://pastebin.com/smz7ECaT